src/management/OrganizationsManager.js

```diff
@@ -0,0 +1,825 @@
+ *
+ * @method    create
+ * @memberOf  module:management.OrganizationsManager.prototype
+ *
+ * @example
+ * management.organizations.create(data, function (err) {
+ *   if (err) {
+ *     // Handle error.
+ *   }
+ *
+ *   // Organization created.
+ * });
+ *
+ * @param   {Object}    data     Organization data object.
+ * @param   {Function}  [cb]     Callback function.
+ *
+ * @return  {Promise|undefined}
+ */
+utils.wrapPropertyMethod(OrganizationsManager, 'create', 'organizations.create');
+
+/**
+ * Get all roles.
+ *
+ * @method    getAll
+ * @memberOf  module:management.OrganizationsManager.prototype
+ *
+ * @example <caption>
+ *   This method takes an optional object as first argument that may be used to
+ *   specify pagination settings. If pagination options are not present,
+ *   the first page of a limited number of results will be returned.
+ * </caption>
+ *
+ * // Pagination settings.
+ * var params = {
+ *   per_page: 10,
+ *   page: 0
+ * };
+ *
+ * management.organizations.getAll(params, function (err, organizations) {
+ *   console.log(organizations.length);
+ * });
+ *
+ * @param   {Object}    [params]          Organizations parameters.
+ * @param   {Number}    [params.per_page] Number of results per page.
+ * @param   {Number}    [params.page]     Page number, zero indexed.
+ * @param   {Function}  [cb]              Callback function.
+ *
+ * @return  {Promise|undefined}
+ */
+utils.wrapPropertyMethod(OrganizationsManager, 'getAll', 'organizations.getAll');
+
+/**
+ * Get an Auth0 organization.
+ *
+ * @method    getByID
+ * @memberOf  module:management.OrganizationsManager.prototype
+ *
+ * @example
+ * management.organizations.getByID({ id: ORGANIZATION_ID }, function (err, role) {
+ *   if (err) {
+ *     // Handle error.
+ *   }
+ *
+ *   console.log(organization);
+ * });
+ *
+ * @param   {Object}    params        Organization parameters.
+ * @param   {String}    params.id     Organization ID.
+ * @param   {Function}  [cb]          Callback function.
+ *
+ * @return  {Promise|undefined}
+ */
+utils.wrapPropertyMethod(OrganizationsManager, 'getByID', 'organizations.get');
+
+/**
+ * Get an Auth0 organization.
+ *
+ * @method    getByName
+ * @memberOf  module:management.OrganizationsManager.prototype
+ *
+ * @example
+ * management.organizations.getByName({ name: ORGANIZATION_NAME}, function (err, role) {
+ *   if (err) {
+ *     // Handle error.
+ *   }
+ *
+ *   console.log(organization);
+ * });
+ *
+ * @param   {Object}    params        Organization parameters.
+ * @param   {String}    params.name   Organization name.
+ * @param   {Function}  [cb]          Callback function.
+ *
+ * @return  {Promise|undefined}
+ */
+utils.wrapPropertyMethod(OrganizationsManager, 'getByName', 'organizationsByName.get');
+
+/**
+ * Update an existing organization.
+ *
+ * @method    update
+ * @memberOf  module:management.OrganizationsManager.prototype
+ *
+ * @example
+ * var data = { name: 'New name' };
```

Comment:
```suggestion
 * var data = { display_name: 'New name' };
```

I get Payload validation error: 'Additional properties not allowed: name' when I try to pass { name: 'foo' } (also 'New name' is not a valid name)